modules/lang-painless/src/main/resources/org/elasticsearch/painless/org.elasticsearch.script.fields.txt

```diff
@@ -10,8 +10,8 @@
 
 # API
 class org.elasticsearch.script.field.Field {
-  org.elasticsearch.script.field.Converter BigInteger
-  org.elasticsearch.script.field.Converter Long
+  org.elasticsearch.script.field.Converter BigInteger @augmented[augmented_canonical_class_name="org.elasticsearch.script.field.BigIntegerField"]
```

Comment:
What's going on here?  I don't see BigInteger on Field.

Reply:
I moved these singletons to their respective field classes. The allowlist in Painless uses the augmented fields for this separation. Note this is consistent with any extension of field outside of core as well like UnsignedLongField.